packages/apps/src/overlays/Attest.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017-2020 @polkadot/apps authors & contributors
```

Comment:
Not the right way to do it to fit in with existing process. The overlays are absolutely horrible. (Cannot get rid of them all atm) 

Add a counter alert to the menu, example -

- https://github.com/polkadot-js/apps/blob/master/packages/page-settings/src/useCounter.ts
- https://github.com/polkadot-js/apps/blob/master/packages/apps-routing/src/settings.ts#L16

Reply:
I would counter-argue that the red notifications on the menu bar are not strong enough for the message we're trying to deliver.

For now, the red notifications are indications that something is happening, as an "info"-level alert (e.g. referendum/proposal are happening right now). It's fine if the user dismisses them (and sometimes will).

On the other hand, attest is a mandatory step: users can't even receive their tokens before attesting. A "warning"/"error"-level alert is needed.

While their UX is indeed not perfect, I do believe that they are better suited, and show immediately the action the users need to take (via the explicit text). "Cannot get rid of them all atm" -> I do see a small "x" button to hide the top bar notification.

Reply:
Sorry, it is just the way it is in the apps UI. Consistency, adjust, please.